twisted/test/test_defer.py

```diff
@@ -2446,3 +2446,23 @@ def test_cancelDeferUntilLockedWithTimeout(self):
         self.assertFalse(timeoutCall.active())
         self.assertIsNone(self.lock._timeoutCall)
         self.failureResultOf(deferred, defer.CancelledError)
+
+
+
+class DeprecationTests(unittest.TestCase):
+    """
+    Test for deprecated code.
+    """
+    def test_deprecatedTimeoutError(self):
+        """
+        Test that L{twisted.internet.defer.TimeoutError} is deprecated.
```

Comment:
I did not find that web page or your comment helpful as part of this review,
in terms of giving me concrete feedback as to what to do.
There are other examples in this file that followed the same docstring convention.
If you want something else, please provide the text that you prefer.